bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapWeatherAndForecastHandler.java

```diff
@@ -298,6 +299,9 @@ private void updateCurrentChannel(ChannelUID channelUID) {
                     state = getQuantityTypeState(weatherData.getSnow() == null ? 0 : weatherData.getSnow().getVolume(),
                             MILLI(METRE));
                     break;
+                case CHANNEL_VISIBILITY:
+                    state = new QuantityType<>(weatherData.getVisibility(), METRE).toUnit(KILO(METRE));
```

Comment:
This breaks the non-null of state, somehow `toUnit` is nullable :-(

Why does this still not break in Maven :-(